mock/py/mockbuild/backend.py

```diff
@@ -477,7 +477,6 @@ def chain(self, args, options, buildroot):
                     resultdir = os.path.join(self.config['local_repo_dir'], pdn)
                     self.buildroot.resultdir = resultdir
                     self.buildroot.resetLogging(force=True)
-                    file_util.mkdirIfAbsent(resultdir)
```

Comment:
Uff. I'm not sure about this. But yeah, in theory it should not be needed. Let be brave and do it.

Reply:
https://github.com/rpm-software-management/mock/blob/8122bda5d159f75f068199b87ae6172325cd436e/mock/py/mockbuild/buildroot.py#L627-L632

This is the force=True case, I think we are safe.